owl/EASE-OBJ.owl

```diff
@@ -1876,6 +1885,16 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#FunctionalDesign -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#FunctionalDesign">
+        <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Design"/>
+        <rdfs:comment>The design of an object from functionality point of view. A functional design is useful to design complex modular devices which assures that each modular component has specific responsibility and can function with minimum side effect on other components of that device. </rdfs:comment>
```

Comment:
still I do not like a sentence like "A design is useful to design" that is not providing any information.

Reply:
I think that "responsibility" is rather used for agents.